install/homebox-install.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/sysadminsmedia/homebox
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Homebox"
+RELEASE=$(curl -s https://api.github.com/repos/sysadminsmedia/homebox/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+wget -q https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz
+tar -xf homebox_Linux_x86_64.tar.gz
+rm -rf homebox_Linux_x86_64.tar.gz
+mkdir -p /opt
```

Comment:
Doesn't `/opt` already exist?

Reply:
right. ive develop on an debian system and delete after every lxc the folder. i remove it